config/locales/account_reset/en.yml

```diff
@@ -33,7 +33,7 @@ en:
       confirm: If you cancel now, you must create a new request and wait another 24
         hours to delete your account.
       header: You requested to delete your account
-      wait_html: There is a 24-hour waiting period to delete your account. In
+      wait_html: There is a %{interval} waiting period to delete your account. In
```

Comment:
Just to confirm, this will now say "24 hours waiting period" vs. the previous "24-hour waiting period"? I don't have a strong feeling about this, but it does seem at odds with prior comments about wanting to preserve the singular "hour".

Reply:
This should use the `%{hour}` instead of `%{interval}`. I can fix that.